arch/arm/src/stm32h7/stm32_fmc.c

```diff
@@ -0,0 +1,196 @@
+/************************************************************************************
```

Comment:
The length of the comments section should be identical throughout the file. 
There is no need for such long lines in this file. The same applies to stm32_fmc.h file.

I see that this looks the same in arch/stm32 - I'll fix that.